tests/test_collection.py

```diff
@@ -14,119 +14,131 @@
 from tests.helpers import assert_raises, extract, generate_col_name
 
 
-def test_collection_attributes(db, col, username):
+def test_collection_attributes(client, col, sys_fabric):
     assert col.context in ['default', 'async', 'batch', 'transaction']
-    assert col.username == username
-    assert col.db_name == db.name
-    assert col.name.startswith('test_collection')
-    assert repr(col) == '<StandardCollection {}>'.format(col.name)
+    assert col.tenant_name == client._tenant.name
+    fabric = client._tenant.useFabric(sys_fabric.name);
+    assert col.fabric_name == fabric.name
+    assert col.name.startswith('test_collection') == True
+    assert repr(col) == f'<StandardCollection {col.name}>'
 
 
-def test_collection_misc_methods(col, bad_col):
+def test_collection_misc_methods(col, sys_fabric, client):
```

Comment:
here it should actually be tst_fabric (since to get sys fabric you will need useFabric("_system") ) plus col collection was created in tst_fabric